src/core/client/admin/components/UserStatus/UserStatusChange.tsx

```diff
@@ -43,7 +49,7 @@ const UserStatusChange: FunctionComponent<Props> = ({
         <ClickOutside onClickOutside={toggleVisibility}>
           <Dropdown>
             {!banned && (
-              <Localized id="community-userStatus-banUser">
+              <Localized id="community-userStatus-ban">
```

Comment:
Ensure you modify the default text (below) to reflect the copy translation change (it still reads `Ban User`).